### PR TITLE
add responsive version

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me

File: frontend/src/components/CardCalendar/index.jsx [hunk 1]
```diff
@@ -19,31 +19,37 @@ export default function CardCalendar({
...
+      <div className="firstPart">
+        <div className="day">
+          <h3>J{day}</h3>
+        </div>
         <img
-          src={`${place === "domicile" ? domicile : exterieur}
-            `}
-          alt="place"
-          className="place"
+          src={`../../src/assets/images/logos/${hit}.png`}
```
Comment (LoicBrassart) [inline frontend/src/components/CardCalendar/index.jsx:27]: Use `@assets`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix the call to your assets

**Review by LoicBrassart — APPROVED** [review]

**Pull request merged**